android/jni/com/mapswithme/maps/Framework.cpp

```diff
@@ -543,21 +543,32 @@ void Framework::EnableDownloadOn3g()
 {
   m_work.GetDownloadingPolicy().EnableCellularDownload(true);
 }
-uint64_t Framework::RequestUberProducts(JNIEnv * env, jobject policy, ms::LatLon const & from,
+uint64_t Framework::RequestTaxiProducts(JNIEnv * env, jobject policy, ms::LatLon const & from,
                                         ms::LatLon const & to,
-                                        uber::ProductsCallback const & callback,
-                                        uber::ErrorCallback const & errorCallback)
+                                        taxi::SuccessfulCallback const & callback,
+                                        taxi::ErrorCallback const & errorCallback)
 {
-  auto const uberApi = m_work.GetUberApi(ToNativeNetworkPolicy(env, policy));
-  if (!uberApi)
+  auto const taxiEngine = m_work.GetTaxiEngine(ToNativeNetworkPolicy(env, policy));
+  if (!taxiEngine)
     return 0;
```

Comment:
At least, there should be some constant, say, kInvalidRequestId or something like that.